static/js/hoc/withChannelTracker_test.js

```diff
@@ -42,11 +48,30 @@ describe("withTracker", () => {
     )
   })
 
-  it("should not call GA create and pageview if channel does not have a tracking id", () => {
+  it("should not call GA create and pageview if channel does not have a tracking id", async () => {
     channel.ga_tracking_id = null
     window.location = "http://fake/c/path"
-    WrappedPage = withChannelTracker(TestPage)
-    renderPage({ location: window.location, channel: channel })
+    await render({}, { location: window.location, channel: channel })
     assert.ok(gaGaStub.notCalled)
   })
+
+  //
+  ;[true, false].forEach(updateChannel => {
+    it(`${shouldIf(
+      updateChannel
+    )} call loadGA() on componentDidUpdate`, async () => {
+      channel.ga_tracking_id = "UA-FAKE-01"
+      window.location = "http://fake/c/path"
+      const { wrapper } = await render(
+        {},
+        { location: window.location, channel: channel }
+      )
+      const prevProps = {
+        location: window.location,
+        channel:  updateChannel ? null : channel
+      }
+      wrapper.instance().componentDidUpdate(prevProps)
+      assert.equal(gaGaStub.callCount, updateChannel ? 4 : 2)
```

Comment:
You can parametrize the `prevProps.channel` value at line 71 as well as this expected call count. Also, it's a little odd that the test case description says that `loadGA()` shouldn't be called in one case, but the assertion says that it should be called twice